src/v/redpanda/admin/api-doc/shadow_indexing.json

```diff
@@ -0,0 +1,25 @@
+"/v1/shadow_indexing/sync_local_state/{topic}/{partition}": {
+  "post": {
+    "summary": "Check content of the bucket",
```

Comment:
Nit: also change the description? “Check content” sounds like something without side effects.

Reply:
updated it